core/os/device/deviceinfo/cc/windows/query.cpp

```diff
@@ -146,7 +146,21 @@ bool createContext(void* platform_data) {
     gContext.mNumCores = sysInfo.dwNumberOfProcessors;
 
     DWORD size = MAX_COMPUTERNAME_LENGTH + 1;
-    GetComputerNameA(gContext.mHostName, &size);
+    WCHAR host_wide[MAX_COMPUTERNAME_LENGTH + 1];
+    if (!GetComputerNameW(host_wide, &size)) {
+        snprintf(gContext.mError, sizeof(gContext.mError),
+                 "Couldn't get host name: %d", GetLastError());
+    }
```

Comment:
I think we want an early-return here. If we host_wide is empty, or garbage, I would be worried about what WideCharToMultiByte would return.